src/main/java/com/bitmovin/api/analytics/query/AnalyticsOperator.java

```diff
@@ -0,0 +1,26 @@
+package com.bitmovin.api.analytics.query;
+
+public enum AnalyticsOperator
+{
+    EQ("EQ"),
+    NE("NE"),
+    LT("LT"),
+    LTE("LTE"),
+    GT("GT"),
+    GTE("GTE"),
+    CONTAINS("CONTAINS"),
+    NOTCONTAINS("NOTCONTAINS");
+
+    private final String text;
+
+    AnalyticsOperator(String text)
+    {
+        this.text = text;
+    }
+
+    @Override
+    public String toString()
```

Comment:
Don't really know if this is necessary to override the toString() method because the String values are the same as the enum definition. But leave it. I think it don't hurt

Reply:
@moser4035 I think if we don't do toString it will lowercase the value? And we need them uppercase in Analytics